### DISCUSSION TITLE
Can i use pangolin 0.5ver to build pangolin_viewer?

### DISCUSSION THREAD

**Discussion by G4419** [stella-cv/stella_vslam#574] [Q&A]
When i use pangolin 0.5ver  to build viewer, i got a error .
I don't know if it's possible to fix this error.
Can i use pangolin 0.5ver to build pangolin_viewer?
![pangolin](https://github.com/stella-cv/stella_vslam/assets/48234708/fc760a41-51e6-43a7-8b14-91092ec820d5)

**Comment by Skwangles** [comment]
https://stella-cv.readthedocs.io/en/latest/installation.html#requirements-for-pangolinviewer
Stella was tested on 0.7+ by the looks of things, so probably not based on your errors.

Is there a specific reason you need to use 0.5?
If so, you can manually edit the pangolin_viewer code yourself and try get it to run, as based on your errors it looks like since 0.5 the names have changed.
